frontend/src/components/widgets/ColorPicker/ColorPicker.test.tsx

```diff
@@ -75,29 +84,41 @@ describe("ColorPicker widget", () => {
     expect(style.width).toBe(getProps().width)
   })
 
-  it("should render a default color", () => {
+  it("should render a default color in the preview and the color picker", () => {
+    expect(wrapper.find(".color-preview").prop("style")).toEqual({
+      backgroundColor: "#000000",
+    })
+
+    wrapper.find(UIPopover).simulate("click")
+
     expect(wrapper.find(ChromePicker).prop("color")).toStrictEqual(
       new String(props.element.get("default"))
     )
   })
 
   it("supports hex shorthand", () => {
+    wrapper.find(UIPopover).simulate("click")
+
     // @ts-ignore
     wrapper.find(ChromePicker).prop("onChangeComplete")({
       hex: "#333",
     })
+    wrapper.update()
```

Comment:
Not sure if this is right but it's the only way I found would work. Please let me know if there's a better solution!

Reply:
We should use shallow rendering and we could solve this with [this enzyme method](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/renderProp.html)